player/command.c

```diff
@@ -6964,6 +6974,8 @@ const struct mp_cmd_def mp_cmds[] = {
 
     { "ab-loop-align-cache", cmd_align_cache_ab },
 
+    { "begin-vo-dragging", cmd_begin_vo_dragging },
```

Comment:
What's the utility in exposing this as a command? I feel like this should just be an internal implementation detail.

Reply:
should work if you bind it, no?

Reply:
I feel like it simulating dragging should be a part of the `mouse` command if it's not already.